### ISSUE TITLE
Monaco editor suggestions for schema 

### ISSUE THREAD

**Issue by sunker** [grafana/redshift-datasource#16]
Related to https://github.com/grafana/redshift-datasource/issues/36

The monaco editor supports adding typeahead suggestions using the [getSuggestions prop](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Monaco/types.ts#L36). We should use this to add suggestions for tables and schemas etc. 

Using the [redshift data api](https://aws.amazon.com/blogs/big-data/using-the-amazon-redshift-data-api-to-interact-with-amazon-redshift-clusters/), we should use `list-schema`, `list-tables` and `describe-table` to load meta data that can be used in suggestions. See [here](https://github.com/grafana/timestream-datasource/blob/e6235789ec0ffae2ffecba8120250bf24449ad90/src/SchemaInfo.ts#L118-L155) how the time stream data source is feeding monaco with suggestions relalated to the schema. 

Before we can start building this in the redshift plugin, we need to make a PR to the sqlds repo to implement methods for tables and schemas. See [this](https://github.com/grafana/sqlds/issues/19#issuecomment-866844604) issue for context. Also, Kevin Minehart from the enterprise squad should be able to support. 

`list-databases` and `list-schemas` takes a database as input. For that, we can use the database that is configured in the [data source json data](https://github.com/grafana/redshift-datasource/blob/cdecce646d06e6669f053ae1369d4221e95fe8ed/pkg/redshift/models/settings.go#L14). Haven't tested this myself, but it seems like it's also possible to do [cross-database queries](https://aws.amazon.com/about-aws/whats-new/2020/10/announcing-cross-database-queries-for-amazon-redshift-preview/). In that case, you're able to query other databases than the one that is specified in data source config page. If so, I think it's fine to not include those data bases in the suggestions that we feed Monaco with.

There is also an api for listing all columns in a table. Even though it would be nice to get suggestions for that in the editor, we don't want to issue one request to `describe-table` upon loading of the query editor since there could potentially be hundreds or thousands of tables. 

At some point I'd like to also provide smart, custom intellisense to the redshift code editor, but that's not part of the MVP. I created [this](https://github.com/grafana/grafana/issues/36061) in the grafana repo to collect some feedback around that.
